newIDE/app/src/UI/Theme/index.js

```diff
@@ -2,29 +2,27 @@
 import { createMuiTheme } from '@material-ui/core/styles';
 import { isLtr } from '../../Utils/i18n/RtlLanguages';
 import memoize from '../../Utils/Memoize';
-import DefaultTheme, {
-  themeName as defaultThemeName,
-  type Theme,
-} from './DefaultTheme';
-import DarkTheme from './DarkTheme';
-import NordTheme from './NordTheme';
-import SolarizedDarkTheme from './SolarizedDarkTheme';
-import './Global.css';
 
-// To add a new theme:
-// * copy the folder of an existing one (DarkTheme for example),
-// * import it at the top of the file
-// * add it below:
-export const themes = {
-  [defaultThemeName]: DefaultTheme,
-  Dark: DarkTheme,
-  Nord: NordTheme,
-  'Solarized Dark': SolarizedDarkTheme,
-};
+import DefaultTheme from './DefaultTheme';
+import { themes as themeList } from './ThemeRegistry';
```

Comment:
Nitpicking: I usually avoid "as" (because if you have to rename something, you might as well rename it in its file) (and this allow to search in the codebase easily for it). Here, let's call it themeList directly in ThemeRegistry :)

Reply:
Ping ;) @randers444